README.md

```diff
@@ -172,14 +175,16 @@ More examples can be found in the **[hack/examples](hack/examples/README.md)** N
     - [Best Practices and Common Pitfalls](/docs/concepts/best-practices-and-common-pitfalls.md)
     - [Architecture](/docs/concepts/architecture.md)
     - Kubernetes
-        - [Invoking Functions by Name with a Kubernetes Ingress](/docs/concepts/k8s/function-ingress.md)
```

Comment:
Please undo this change and keep indentation under `Kubernetes` so it wont stand dangling.